examples/compare_CPMG_echo.py

```diff
@@ -141,18 +141,21 @@
         )
         thisd.squeeze()
         fl.next("raw data for %s" % thislabel)
-        rows = np.prod([thisd.shape[j] for j in thisd.dimlabels[:-1]])
-        if rows < 500:
-            fl.image(thisd)
-        else:
-            fl.image(thisd, interpolation="bilinear")
+        fl.image(thisd, interpolation='auto')
+        fl.next("raw data for %s, slice overall and signal average" % thislabel)
+        fl.image(thisd['ph_overall',-1].sum('nScans').ift('t2'), interpolation='auto')
```

Comment:
since `ph_overall` will always be -1

Reply:
I actually ended  up undoing this, b/c I wanted to see if there were artefacts for other ph_overall 

you can resolve this comment once you read it